pymc/tests/test_distributions_random.py

```diff
@@ -1176,6 +1176,41 @@ class TestDirichlet(BaseTestDistribution):
     ]
 
 
+class TestStickBreakingWeights(BaseTestDistribution):
+    pymc_dist = pm.StickBreakingWeights
+    pymc_dist_params = {"alpha": 2.0, "K": 19}
+    expected_rv_op_params = {"alpha": 2.0, "K": 19}
+    sizes_to_check = [None, 17, (5,), (11, 5), (3, 13, 5)]
+    sizes_expected = [
+        (20,),
+        (17, 20),
+        (
+            5,
+            20,
+        ),
+        (11, 5, 20),
+        (3, 13, 5, 20),
+    ]
+    tests_to_run = [
+        "check_pymc_params_match_rv_op",
+        "check_rv_size",
+        "check_basic_properties",
+    ]
+
+    def check_basic_properties(self):
+        default_rng = aesara.shared(np.random.default_rng(1234))
+        draws = pm.StickBreakingWeights.dist(
+            alpha=3.5,
+            K=19,
+            size=(2, 3, 5),
+            rng=default_rng,
+        )
+
+        assert np.allclose(draws.sum(-1).eval(), 1)
+        assert np.all(draws.eval() >= 0)
+        assert np.all(draws.eval() <= 1)
```

Comment:
No need to call eval 3 times, you can store the results in a variable and then do the 3 checks on them

Reply:
Are you saying to do something like: `assert all([bool, bool2, bool3])`

Reply:
Oops, got it!